lib/rubocop/cop/betterment/use_global_strict_loading.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Betterment
+      module UseGlobalStrictLoading
+        # This cop ensures that `self.strict_loading_by_default = <any value>` is not set in ActiveRecord models.
+        class ByDefaultForModels < Base
+          extend AutoCorrector
+          include RangeHelp
+
+          MSG = 'Do not set `self.strict_loading_by_default` in ActiveRecord models.'
+
+          # @!method strict_loading_by_default_set?(node)
+          def_node_matcher :strict_loading_by_default_set?, <<~PATTERN
+            $(send self :strict_loading_by_default= _)
+          PATTERN
+
+          def safe_autocorrect? = false
```

Comment:
I don't think this method is intended to be overridden. Consumers of this cop should be able to override the defaults that we're providing in `config/default.yml`.

https://github.com/rubocop/rubocop/blob/35c36774f0e79dce0b486d9a0a89e5c3f80ed478/spec/rubocop/cop/cop_spec.rb#L516-L536